.github/workflows/test.yml

```
@@ -0,0 +1,35 @@
+  contents: read
+  actions: read
+  checks: write
+
+jobs:
+  run-common-tests:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)